data/summary/test/test_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package test_test
```

Comment:
You're breaking the convention here - the `test` package ought to contain only test helper functions (e.g. to generate data), not actual Ginkgo Tests